ts/features/itwallet/machine/eid/__tests__/machine.test.ts

```diff
@@ -1758,4 +1761,54 @@ describe("itwEidIssuanceMachine", () => {
 
     expect(actor.getSnapshot().value).toStrictEqual("Success");
   });
+
+  it("should not call navigateToExtendedLoadingScreen before 5000ms in TrustFederationVerification state", async () => {
```

Comment:
What do you think about adopting the Arrange → Act → Assert pattern for these tests?
It would likely improve their structure and readability. For instance, we could Arrange (set up mocks) first, then Act (start the flow with the actors), and finally Assert (verify expectations).

```
// Arrange
mockFoo.mockImplementation(...);

// Act
actor.start();

// Assert
expect(foo123).toHaveBeenCalledTimes(expected);
```